listener/kcp/manager_factory_test.go

```diff
@@ -25,6 +27,9 @@ func TestNewManager(t *testing.T) {
 		"successful_manager_creation":     {isKCPEnabled: false, expectErr: false},
 	}
 
+	log, err := logger.New(logger.DefaultConfig())
```

Comment:
Just as an information in case you are not aware: we also have a `testlogger` package that can be used to actually check log messages (not necessary here)

Reply:
Good to know, thanks!